src/components/user/UserRepo.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import User from './User.jsx';
+
+const Repo = ({ repos }) => (
+ const repoElements = repoElements.map(repo => (
+   <li key={Character.id}>
+     <User {...user} />
+    </li>
+ ));
+Repo.propTypes = {
+  name: PropTypes.string.isRequired,
```

Comment:
If you're going to be using name and repos_url as props you'll have to pass them through with repos on line 5!